Dockerfile.rpm

```diff
@@ -10,13 +10,15 @@ ARG REDHAT_USERNAME
 ARG REDHAT_PASSWORD
 ARG RHEL="false"
 
-RUN if [ "$RHEL" = "true" ] ; then subscription-manager register --username ${REDHAT_USERNAME} --password ${REDHAT_PASSWORD} --auto-attach ; fi
-RUN if [ "$RHEL" = "true" ] ; then yum-config-manager --enable 'rhel-*-server-optional-rpms' ; fi
+RUN if [ "$RHEL" = "true" ] ; then subscription-manager register --username ${REDHAT_USERNAME} --password ${REDHAT_PASSWORD} ; fi
+RUN if [ "$RHEL" = "true" ] ; then subscription-manager refresh ; fi
+RUN if [ "$RHEL" = "true" ] ; then subscription-manager attach --auto ; fi
+RUN if [ "$RHEL" = "true" ] ; then command -v yum-config-manager && yum-config-manager --enable 'rhel-*-server-optional-rpms' || true ; fi
```

Comment:
RHEL8 doesn't have `yum-config-manager` and doesn't need it so check if the command exists before running it